### PR TITLE
[DHP-917] Add meta tags to prevent search engines from indexing connected devices page

### PR DESCRIPTION
**Pull request by mariahsager**
## Summary

- Added meta tags to our page (health-care/connected-devices) to prevent search engines from indexing it

## Testing done

- Ran unit tests
- Ran page locally and inspected source to verify that the meta tags were there

## What areas of the site does it impact?

- Only our page is affected

### Quality Assurance & Testing

- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated

### Error Handling

- [x] Browser console contains no warnings or errors.

### CODE REVIEW

File: src/applications/dhp-connected-devices/tests/containers/App.spec.jsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ describe('App', () => {
       </Provider>,
     );
 
-    expect(wrapper.find('va-loading-indicator').length).to.equal(0);
+    // expect(wrapper.find('va-loading-indicator').length).to.equal(0);
```
Comment (yonidoronpeters) [inline src/applications/dhp-connected-devices/tests/containers/App.spec.jsx:58]: Can we please remove this line if it's not needed?

File: src/applications/dhp-connected-devices/containers/App.jsx [hunk 2]
```diff
@@ -16,6 +17,9 @@ export default function App() {
 
   return (
     <div className="usa-grid-full margin landing-page">
+      <MetaTags>
+        <meta name="robots" content="noindex" />
+      </MetaTags>
```
Comment (yonidoronpeters) [inline src/applications/dhp-connected-devices/containers/App.jsx:22]: Is there a way to add a test that MetaTags exist?

File: src/applications/dhp-connected-devices/tests/containers/App.spec.jsx (hunk 1 above)
Comment (mariahsager-tw) [reply src/applications/dhp-connected-devices/tests/containers/App.spec.jsx:58]: Removed!

File: src/applications/dhp-connected-devices/containers/App.jsx (hunk 2 above)
Comment (jpresley23-tw) [reply src/applications/dhp-connected-devices/containers/App.jsx:22]: We thought about that but it seems like all that would do is test the library. We're importing `MetaTags` from `react-meta-tags` library. Should we still run a test for that?

Comment (yonidoronpeters) [reply src/applications/dhp-connected-devices/containers/App.jsx:22]: I think we can still test that the MetaTags exists on the page without testing its functionality. So if someone accidentally removes it, we have a breaking test that will let them know something isn't right

Comment (mariahsager-tw) [reply src/applications/dhp-connected-devices/containers/App.jsx:22]: We gave up and created a card DHP-939 to write a test later

**Review by yonidoronpeters — APPROVED** [review]

**Pull request merged**